### PR TITLE
Added support for Meteor developer accounts including minimal test

### PR DESCRIPTION
**Pull request by neopostmodern**
This adds support for Meteor developer accounts.
I had to rewrite a bunch of stuff to be based on arrays now, because Meteor developer accounts expose multiple email accounts.
Added another test for meteor then too.

### CODE REVIEW

**Comment by neopostmodern** [comment]
FYI the tests fail because of some package naming issue (I didn't touch that) but they ran locally after renaming the folder. You could change the package.js to fix this.

**Comment by splendido** [comment]
Hi @neopostmodern,
I'll try to have a look at your work this week!

Thanks!

**Comment by neopostmodern** [comment]
@splendido - any news on this? Something I'd need to patch?

**Comment by splendido** [comment]
I'm sorry @neopostmodern, I had no time to look into this :(
...it might be I'll have a bit of time during next days, but I cannot promise

File: lib/accounts-emails-field.js [hunk 1]
```diff
@@ -32,28 +32,45 @@ var whitelistedServices = ['facebook', 'linkedin'];
...
+    if (service.emailAddress)
+      emails = [ { address: service.emailAddress } ];
+    if (service.emails)
+      emails = service.emails;
+
+    return emails.map(function (email) {
+      if (!email.address)
+          // e.g. GitHub provides a null value in the field "email" in case the email
+          //      address is not verified!
+          return {address: null, verified: false};
```
Comment (splendido) [inline lib/accounts-emails-field.js]: isn't this potentially a problem?
I'd rather skip the `null` address...

Comment (splendido) [inline lib/accounts-emails-field.js]: oh, my bad. Sorry!
I'll try to revise it a little then...

**Comment by splendido** [comment]
Tnx!

File: lib/accounts-emails-field.js (hunk 1 above)
Comment (splendido) [inline lib/accounts-emails-field.js]: ...actually it is skipped [here](https://github.com/splendido/meteor-accounts-emails-field/blob/master/lib/accounts-emails-field.js#L95)

**Comment by splendido** [comment]
Published version 1.1.0!


Let me know,
Luca

**Comment by splendido** [comment]
See [Advanced API](https://github.com/splendido/meteor-accounts-emails-field/#advanced-api) and [v1.2.0](https://github.com/splendido/meteor-accounts-emails-field/releases/tag/v1.2.0)

Thanks again!

**Pull request merged**